samples/mcp-integration-with-kb/README.md

```diff
@@ -0,0 +1,82 @@
+# MCP Integration with Amazon Bedrock Knowledge Bases
+
+This repository outlines a basic implementation of the [Model Context Protocol](https://modelcontextprotocol.io/) integration with Amazon Bedrock Knowledge Bases
+
+## Overview
+
+There are two parts to this implementation:
+
+1. The `user_interfaces/chat_bedrock_st.py` file, which handles the Streamlit/User Interface for the chatbot
+2. The `client_server.py` file, which handles the MCP client and server implementation
+
+The exact MCP server code leveraged can be found in the [src/bedrock-kb-retrieval-mcp-server](../../src/bedrock-kb-retrieval-mcp-server/) folder.
+
+### Architecture
+
+![Architecture](./assets/simplified-mcp-flow-diagram.png)
+
+## Setup
+
+### Prerequisites
+
+- The [uv](https://docs.astral.sh/uv/getting-started/installation/) package manager
+- AWS Account with Bedrock access and proper IAM permissions
```

Comment:
Consider a link to the [Getting started with Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/getting-started.html)